src/context/user-context.js

```diff
@@ -0,0 +1,32 @@
+import { createContext, useContext, useState } from "react";
+import { useCookieHandler } from "custom-hooks";
+
+const userInitialState = {
+  isUserAuthTokenExist: false,
+  userAuthToken: "",
+};
+
+const UserContext = createContext({
```

Comment:
Here you should write it in camel case as using capital first letter naming convention is used for components naming so this can be misunderstood as a component.

Reply:
But, I think for context too we use the pascal case. See this https://reactjs.org/docs/context.html#reactcreatecontext

Reply:
Thanks for correcting me here. Was not aware of this. Appreciate it.